rfs/root.go

```diff
@@ -53,7 +60,24 @@ func (r *rfsRoot) Create(ctx context.Context, name string, flags uint32, mode ui
 		return nil, nil, 0, PermissionDenied
 	}
 
-	return r.LoopbackNode.Create(ctx, name, flags, mode, out)
+	flags = flags &^ syscall.O_APPEND
+	fd, err := syscall.Open(p, int(flags)|os.O_CREATE, mode)
+	if err != nil {
+		return nil, nil, 0, fs.ToErrno(err)
+	}
+
+	st := syscall.Stat_t{}
+	if err := syscall.Fstat(fd, &st); err != nil {
```

Comment:
err was already declared, you can use `=` instead

Reply:
Done :eyes: